app/modules/users/components/inventory_or_listing_nav.svelte

```diff
@@ -1,47 +1,50 @@
 <script>
   import { i18n } from '#user/lib/i18n'
-  import app from '#app/app'
   import { isOpenedOutside } from '#lib/utils'
-  import { isNumber } from 'underscore'
+  // import { isNumber } from 'underscore'
 
-  export let userModel, groupModel, currentSection
+  export let user, group, currentSection
```

Comment:
Manipulating Backbone model was on purpose, in order to know what we are dealing with here. The idea was to only call models when necessary (the lastet possible), to not depend on models too much.

Reply:
You may ask :stuck_out_tongue_winking_eye:

Reply:
We are getting rid of Backbone, so if there is less backbone model, shouldn't it be better? What Backbone-model-free solution would you prefer? There is indeed some implicit here, which is that we are dealing with a serialized user object or group object (as they are expected to have `inventoryPathname` and `listingsPathname` attributes), but that seems acceptable to me. We could get much more explicit by typing our component interfaces with Typescript, but we are not there yet.

Reply:
It was on purpose to explicitly stating it's a bockbone model. But yes, its better without it.